beta-pipelines/methylome/snM3C.wdl

```diff
@@ -0,0 +1,177 @@
+version 1.0
+
+workflow snM3C {
+
+  input {
+    # demultiplexing inputs
+    Array[File] fastq_input_read1
+    Array[File] fastq_input_read2
+    File random_primer_indexes
+    String plate_id
+    String output_basename = plate_id
+
+    # mapping inputs
+    File tarred_index_files
+    File mapping_yaml
+    File snakefile
+    File chromosome_sizes
+    File genome_fa
+  }
+
+  call Demultiplexing {
+    input:
+      fastq_input_read1 = fastq_input_read1,
+      fastq_input_read2 = fastq_input_read2,
+      random_primer_indexes = random_primer_indexes,
+      plate_id = plate_id
+    }
+
+  call Mapping {
+    input:
+      tarred_demultiplexed_fastqs = Demultiplexing.tarred_demultiplexed_fastqs,
+      tarred_index_files = tarred_index_files,
+      mapping_yaml = mapping_yaml,
+      snakefile = snakefile,
+      chromosome_sizes = chromosome_sizes,
+      genome_fa = genome_fa,
+      plate_id = plate_id
+    }
+
+  output {
+    File MappingSummary = Mapping.mappingSummary
+    File allcFiles = Mapping.allcFiles
+    File allc_CGNFiles = Mapping.allc_CGNFiles
+    File bamFiles = Mapping.bamFiles
+    File detail_statsFiles = Mapping.detail_statsFiles
+    File hicFiles = Mapping.hicFiles
+  }
+}
+
+task Demultiplexing {
+  input {
+    Array[File] fastq_input_read1
+    Array[File] fastq_input_read2
+    File random_primer_indexes
+    String plate_id
+
+    String docker_image = "ekiernan/yap_hisat:v4"
```

Comment:
@nikellepetrillo Did you update this docker? Would you be able to put in the new docker here?

Reply:
update, it passed with the new docker 👍